ldi-core/ldi-common/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldi/rdf/formatter/JsonLdFrameWriter.java

```diff
@@ -21,11 +19,10 @@ public JsonLdFrameWriter(LdiRdfWriterProperties properties) {
 	}
 
 	@Override
-	public String write(Model model) {
-		return RDFWriter.source(addPrefixesToModel(model))
+	public RDFWriterBuilder builder() {
```

Comment:
The LdiRdfWriter is a strategy pattern where different rdf formats have their own implementation. Given you have a specific rdf format -> binary, I wouldn't expect any changes in the jsonLdWriter